streamd/gstreamer_pipeline.go

```diff
@@ -182,7 +183,7 @@ func newPipeline(d *daemonConfig) (*pipeline, error) {
 		return nil, err
 	}
 
-	p.srtCompositorSink, err = newSRTSink("sink_combined", d.listenCombSRT)
+	p.srtCompositorSink, err = newSRTSink("sink_combined", fmt.Sprintf("srt://%s:%s", d.listenAddr, d.combPort))
```

Comment:
What about the query parameter `mode` in the URI (i.e. `srt://[::]:7002?mode=listener`)?. This is optional when using iPv4, but mandatory with iPv6. The change breaks iPv6 support.

Additionally, you want to wrap d.listenAddr in brackets if it is iPv6.

It is a bit confusing, but here is a good guide: https://github.com/hwangsaeul/libsrt/blob/master/docs/srt-live-transmit.md#:~:text=srt%3A//%5B%3A%3A%5D%3A5000%3Fmode%3Dlistener%20defines%20listener%20mode%20with%20IPv6.%20If%20the

Reply:
Thanks, I was unaware. 

Are we fine with going `?mode=listener` as default and non-configurable for all streams to keep it simple?


> Additionally, you want to wrap d.listenAddr in brackets if it is iPv6.

Good catch!